docs/tutorials/fundamentals/quick_start.md

```diff
@@ -130,7 +130,7 @@ viewer = napari.Viewer()
 
 from skimage import data
 viewer.add_image(data.cell(), name='cell')
-nbscreenshot(viewer)
+nbscreenshot(viewer, alt_text=" ")
```

Comment:
+nbscreenshot(viewer, alt_text="image of a single cell opened in napari viewer")